Sources/ChangelogProducerCore/StringExtensions.swift

```diff
@@ -14,9 +14,47 @@ extension String {
     func pullRequestIDs() -> [Int] {
         return []
     }
-
-    /// Extracts the resolved issue from a Pull Request body.
-    func resolvingIssue() -> Int? {
-        return nil
+    
+    /// Extracts the resolved issues from a Pull Request body.
+    func resolvingIssues() -> [Int] {
+        var resolvedIssues = Set<Int>()
+        
+        let splits = split(separator: "#")
+        
+        let issueClosingKeywords = [
+            "close ",
+            "closes ",
+            "closed ",
+            "fix ",
+            "fixes ",
+            "fixed ",
+            "resolve ",
+            "resolves ",
+            "resolved "
+        ]
+        
+        for (index, split) in splits.enumerated() {
+            let lowerCaseSplit = split.lowercased()
+            
+            for keyword in issueClosingKeywords {
+                if lowerCaseSplit.hasSuffix(keyword) {
```

Comment:
spy = on
I guess you want be able to retrieve list of resolved issues if the user have something similar to : this PR closes issues #1 and #2 and #3.
spy = off

Reply:
Thanks for the feedback @a6delali! 
This is something I had taken into consideration. As far as I know, Github doesn't support this syntax, so it's not something we want to support.
I'll double check to make sure that what I said is true though.

Reply:
It's confirmed here: https://help.github.com/en/github/managing-your-work-on-github/closing-issues-using-keywords
> To close multiple issues, preface each issue reference with one of the above keywords. You must use the keyword before each issue you reference for the keyword to work.